site/docs/components/tabs.mdx

```diff
@@ -0,0 +1,115 @@
+
+- Anatomy:
+    - Tab list: This refers to the container holding each of the tab items.
+    - Selected tab: The current, active tab. One tab—and only one tab—is always selected for a given tab list.
+    - Unselected tab: The other available tabs.
+    - Tab item:
+        - A tab item contains a large clickable target area, a label, a selected tab indicator (when the tab is selected), and optional metadata.
+        - Tab link: The large clickable area of the tab item uses a link to navigate to that tab's content.
+        - Tab label: The label describes what content will be displayed. Use 1 short noun or noun phrase.
+        - Selected tab indicator: This contains a selected tab indicator to show that the tab is current.
+        - Optional metadata: You might display 1 item of metadata, such as a [Badge](/components/badge) to bring data forward from the content in that section or a [Tag](/components/tag) to add supplementary information, such as "New".
+    - Tab content: The content that lives "inside" a tab and is revealed when that tab item is selected.
+    - Separator: To separate tabs from content, use whitespace or the edge of another component, such as Title Blocks or Cards, not a [Divider](/components/divider).
+- Styling:
+    - Align navigation tabs and section tabs left (in left-to-right languages) and never center within a page or section.
+    - For small screens, tab items must be presented without loss of information or functionality, and without requiring scrolling in two dimensions for vertical scrolling content at a width equivalent to 320 CSS pixels and horizontal scrolling content at a height equivalent to 256 CSS pixels, as per [W3C: Understanding Success Criterion 1.4.10: Reflow](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html).
+    - Limit tabs to a maximum of 8 tab items and aim for 3–6. Avoid only 2 tabs.
+    - Avoid using icons in tab items.
+    - Tab content always appears below horizontal tabs. Tab content always appears to the right of vertical tabs in left-to-right languages and to the left of vertical tabs in right-to-left languages.
+- Tab content and ordering:
+    - The first tab is selected by default. The default tab is most important use case of the page
+    - Use smart defaults for defining tabs that accelerate users to the content they care about most. For example, you might surface the most common pathway or personalized content, such as the current user's saved comments.
+    - Avoid tabs that you can "toggle off". That is, 1 tab is always selected. If you want a tab list with no selected tabs, consider instead adding a tab representing "All" the content as the first tab.
+- Navigation:
+    - A user can navigate between tabs to reveal that tab's content by clicking or tapping on the tab item. We don't support swiping between sections using the tab content area (for example, using edge swipes on mobile devices), especially when tabs have content of different length.
```

Comment:
Break up the sentence at ", ". Rewrite " especially when tabs have content of different length." If we ever supported swiping between tabs, we would only support swiping the tab list itself, not the content area below tabs.